### PR TITLE
Dedicated crate for protobuf_runtime

### PR DESCRIPTION
**Pull request by reznikmm**
Move `protobuf_config.gpr` from `config/` to `gnat/` to keep all project files in one folder.

### CODE REVIEW

File: gnat/protobuf_config.gpr [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+   Alire_Host_OS := "linux";
+
+   Alire_Host_Arch := "x86_64";
+
+   Alire_Host_Distro := "ubuntu";
+   Ada_Compiler_Switches := External_As_List ("ADAFLAGS", " ");
+   Ada_Compiler_Switches := Ada_Compiler_Switches &
+          (
+            "-O0"
```
Comment (mgrojo) [inline gnat/protobuf_config.gpr:38]: Should the distribution mode (when the user might even build without Alire) be the development mode or the release mode? I think it makes more sense to distribute the crate config in the release mode. I don't know if the Alire authors propose something about it.

File: alire.toml [hunk 2]
```diff
@@ -19,3 +24,12 @@ prepend = "${CRATE_ROOT}/.objs/compiler"
 
 [build-switches]
 "*".ada_version = "Ada12"
+development.optimization = ["-O0"]
```
Comment (mgrojo) [inline alire.toml:27]: Just out of curiosity, do you override the default used by Alire (`-Og`) for a particular reason?

File: gnat/protobuf_config.gpr (hunk 1 above)
Comment (reznikmm) [reply gnat/protobuf_config.gpr:38]: Sounds reasonable.

File: alire.toml (hunk 2 above)
Comment (reznikmm) [reply alire.toml:27]: I found debugging difficult with `-Og` sometimes. The compiler eliminates some variables and gdb can't show their values.

**Pull request merged**